.travis.yml

```diff
@@ -42,7 +44,6 @@ env:
 matrix:
   allow_failures:
     - env: ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_REPONAME=industrial_core PRERELEASE_DOWNSTREAM_DEPTH=0
-    - env: ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_REPONAME=ros_canopen
```

Comment:
First and foremost, I was wrong in starting this discussion at this line. So +1 for removing this line.
(This particular line is removed only because [the corresponding line is also removed](https://github.com/ros-industrial/industrial_ci/pull/83/files#diff-354f30a63fb0907d4ad57269548329e3L27), totally irelevant from the discussion above. My bad.)